includes/class-fs-plugin-updater.php

```diff
@@ -401,6 +486,18 @@ function get_update_details( FS_Plugin_Tag $new_version ) {
                 );
             }
 
+            if ( $this->_fs->is_premium() ) {
+                $latest_tag = $this->_fs->_fetch_latest_version( $this->_fs->get_id(), false );

```

Comment:
@fajardoleo I think that I'm missing something... based on the API PR, the endpoint was enriched with a new `readme` param that defaults to `false`. It needs to be `true` to return the readme data. I don't see any changes to `_fetch_latest_version()` so I don't understand how it will ever return the data with the readme.

Reply:
@vovafeldman, `_fetch_latest_version()` calls `_get_latest_version_endpoint()` which is responsible for the inclusion of the `readme` param.